topics/spacevim/index.md

```diff
@@ -0,0 +1,16 @@
+---
+aliases: space vim
+created_by: ShiDong Wang
+display_name: SpaceVim
+github_url: https://github.com/SpaceVim/SpaceVim
+logo: spacevim.png
+released: November 6, 2017
+short_description: SpaceVim is a community-driven vim distribution that seeks to provide layer feature.
```

Comment:
I'm not sure I understand "layer feature." What about using the intro from http://spacevim.org/about/?

> SpaceVim is a community-driven vim distribution that supports vim and Neovim.

Reply:
Layer feature is the main feature of SpaceVim. and It is the main difference between it and other vim distribution.